app/docker/views/containers/logs/containerLogsController.js

```diff
@@ -14,6 +15,10 @@ function ($scope, $transition$, $interval, ContainerService, Notifications) {
     }
   };
 
+  $scope.displayTimestamp = function(timestampStatus) {
```

Comment:
No need for a function to update the value of the `state.displayTimestamps` variable. As it's binded using double way binding in the `logViewer` component (`=` binding type), it's going to be updated in here when we change the value in the `logViewer` component.

You can remove this function.